src/Utilities.jl

```diff
@@ -5,6 +5,66 @@ module Utilities
 """
 Utilities
 
+export isexported, iscategory
+
+import Docile.Collector: QualifiedSymbol, Aside
+
+import Docile.Utilities: isexpr
+
+import Docile: Cache
+
 using Compat
 
+convert(::Type{UTF8String}, sym::Symbol)    = utf8(string(sym))
+convert(::Type{UTF8String}, s::ASCIIString) = utf8(s)
+
+function utf8checked!(d::Dict{Symbol, Any}, k)
+    isa(d[k], UTF8String) ? d[k] : (d[k] = convert(UTF8String, d[k]); d[k])
+end
+
+"""
+Get the ``Symbol`` representing an object such as ``Function`` or ``Method``.
+"""
+nameof(s::Symbol) = s
+nameof(m::Module, obj::Symbol)            = obj
+nameof(m::Module, obj::Module)            = module_name(obj)
+nameof(m::Module, obj::Method)            = nameof(Cache.getmeta(m, obj)[:code])
```

Comment:
seems that method is supposed to do what I indented. [name and module reflection](https://github.com/JuliaLang/julia/blob/master/base/reflection.jl#L4)

Leave it for now.